docs/compose/treafik/traefik.toml

```diff
@@ -0,0 +1,33 @@
+# This is just boilerplate stuff you probably have in your own config
+logLevel = "INFO"
+defaultEntryPoints = ["https","http"]
+
+[entryPoints]
+  [entryPoints.http]
+  address = ":80"
+    [entryPoints.http.redirect]
+    entryPoint = "https"
+  [entryPoints.https]
+  address = ":443"
+  [entryPoints.https.tls]
+
+[docker]
+endpoint = "unix:///var/run/docker.sock"
+watch = true
+exposedByDefault = false
+
+# Make sure we get acme.json saved, and onHostRule enabled
+[acme]
+email = "root@mail.doma.in"
```

Comment:
Don't know if Traefic supports variables for its toml files?

Reply:
Unfortunately I couldn’t figure out how — to the best of my knowledge, it’s not possible.
While adding a templater would be possible, I think the common deployment scenario is to have traefik already set-up outside of mailu, and this already configured. Maybe I should make this clearer here?